bin/filter_ambient_rna.r

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env Rscript
+
+library(SoupX)
+library(DropletUtils)
+
+# Load data and estimate soup profile
+soup_object <- load10X(getwd())
+
+# Estimate level of background contamination
+soup_object <- autoEstCont(soup_object)
+
+# Adjust counts accordingly
+output <- adjustCounts(soupx_object)
+
+# Glob for the features TSV, which containes gene IDs and symbols.
```

Comment:
This little commented-out section doesn't work in my current environment. Something is wrong with the dropletutils package's way of creating an H5 file, so in the subsequent step (mimic_cellranger), I just make a .h5 file from the directory using scanpy.